### PR TITLE
Add Random Data Generator Extension

### PR DESCRIPTION
**Pull request by loris**
## Description

Simple extension which use the awesome [Faker.js library](https://github.com/marak/faker.js) to generate massive amount of random/fake data: lorem ipsum, names, avatar images, phone numbers, etc

- Fast and offline (data are generated locally)
- Actions to copy to clipboard, paste to frontmost app, open url in browser
- Frequently used data can be pinned to top of the list
- It supports many locales (default to english)

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/8362/139554233-8a629c2d-3218-4316-8fc5-19f66fff016d.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

**Comment by loris** [comment]
Note I'm unsure about the name of the extension/command. I first went with "Faker" which is cooler but less explicit

File: extensions/random-data-generator/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+const items = buildItems('');
+const groupedItems = _.groupBy(items, 'section');
+
+function FakerListItem(props: { item: Item; pin?: Pin; unpin?: Pin }) {
+  const { item, pin, unpin } = props;
+  const [value, setValue] = useState(item.value);
+
+  return (
+    <List.Item
+      key={item.id}
```
Comment (vimtor) [inline extensions/random-data-generator/src/index.tsx:79]: This `key` is no needed here 😄

Comment (pernielsentikaer) [reply extensions/random-data-generator/src/index.tsx:79]:
```suggestion
```

key is not a property in List.Item - as @vimtor stated, just remove it 😄

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @loris 👋 

That's an awesome plugin! I couldn't find anything else than the key that should be removed, after that, it should be ready to get merged 🚀  Thanks for another great contribution 💯 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/random-data-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+  "description": "Generate random data using Faker library",
+  "icon": "random.png",
+  "author": "loris",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Generate Random Data",
+      "subtitle": "",
+      "description": "Generate random data using Faker library",
```
Comment (PitNikola) [inline extensions/random-data-generator/package.json:13]:
```suggestion
      "title": "Generate Random Data",
      "description": "Generate random data using Faker library",
```

**Comment by PitNikola** [comment]
Cool one!

**Pull request merged**